src/demo-app/drag-and-drop/drag-and-drop-demo.html

```diff
@@ -0,0 +1,19 @@
+<div class="list">
+  <h2>To do</h2>
+  <cdk-drop #one="cdkDrop" [connectedTo]="[two]">
+    <ng-container *ngFor="let item of todo">
+      <div *cdkDrag>{{item}}</div>
+    </ng-container>
+  </cdk-drop>
+</div>
+
+<div class="list">
+  <h2>Done</h2>
+  <cdk-drop #two [connectedTo]="[one]">
+    <ng-container *ngFor="let item of done">
+      <div *cdkDrag>{{item}}</div>
```

Comment:
What happens if your list of items changes after dragging? 

One thing I'm not sure was mentioned in the doc was what's the advantage of having the drag/drop code move around views instead of only firing events that the user would use to update their data model? The latter seems like it would lead to a much simpler API because you wouldn't need an extra layer of templatization.

Reply:
That's one thing that I was hoping would be addressed by using the views, but it wasn't. 

I'm not sure  whether only dispatching events would make it that much simpler to consume, because the user would still have to implement at least three methods for it to work (`insert`, `delete` and `move`). Also I'm not sure how that would work with having the ability to have a custom drop target (placeholder element that is shown in the place where the draggable item would be dropped), because we need to be able to determine which template goes into which view slot.